tests/cypress/integration/regTest_for_export_task/innerExportTask.js

```diff
@@ -0,0 +1,138 @@
+// Start writing your Cypress tests below!
+// If you're unfamiliar with how Cypress works,
+// check out the link below and learn how to write your first test:
+// https://on.cypress.io/writing-first-test
+
+/// <reference types="cypress" />
+
+Cypress.Commands.add('exportInnerTask', ({
+    taskName,
+    archiveCustomName,
+    order,
+    imageClick = false,
+}) => {
+    const orderId = Number.isNaN(parseInt(order, 10)) ? 0 : parseInt(order, 10);
+    const archFileNameFin = `${orderId}_${archiveCustomName}_${taskName}`;
+    cy.get('.cvat-task-page-actions-button')
+        .trigger('mouseover');
+    cy.get('.ant-dropdown')
+        .not('.ant-dropdown-hidden')
+        .within(() => {
+            cy.contains('[role="menuitem"]', new RegExp('^Export task dataset$')).click();
+        });
+    cy.get('.cvat-modal-export-task')
+        .should('be.visible')
+        .find('.cvat-modal-export-select')
+        .click();
+    cy.get('.cvat-modal-export-option-item')
+        .eq(orderId)
+        .click();
+    cy.get('.cvat-modal-export-task')
+        .find('.cvat-modal-export-filename-input')
+        .type(archFileNameFin);
+
+    if (imageClick) {
+        cy.get('.cvat-modal-export-save-images').click();
+    }
+
+    cy.get('.cvat-modal-export-task').contains('button', 'OK').click();
+
+    cy.get('.cvat-notification-notice-export-task-start').should('be.visible');
+    cy.closeNotification('.cvat-notification-notice-export-task-start');
+    cy.getDownloadFileName().then((file) => {
+        cy.verifyDownload(file);
+        cy.get('.ant-notification-notice-info').first().should('be.visible');
+        cy.closeNotification('.ant-notification-notice-info');
+    });
+});
+
+context('Export task inner.', { browser: '!firefox' }, () => {
+    const caseId = '999';
+    const labelName = 'block';
+    const taskName = `Case test inner ${caseId}`;
+    const attrName = 'color';
+    const textDefaultValue = 'red';
+    const imagesCount = 1;
+    const imageFileName = `image_${taskName.replace(/\s+/g, '_').toLowerCase()}`;
+    const width = 800;
+    const height = 800;
+    const posX = 10;
+    const posY = 10;
+    const color = 'gray';
+    const archiveName = `${imageFileName}.zip`;
+    const archivePath = `cypress/fixtures/${archiveName}`;
+    const imagesFolder = `cypress/fixtures/${imageFileName}`;
+    const directoryToArchive = imagesFolder;
+    const newLabelName = 'person';
+    const exportOrders = [0, 1, 2];
```

Comment:
Choosing an export format by its order is not robust option I think. Order can be changed easily. 
Better to choose by name